spec/features/book_index_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Book index', type: :feature do
+  context 'when there are no books on the' do
+    let(:nothing) { 'Nothing here yet' }
+
+    before(:each) do |test|
+      create(:book, title: 'JO', category: 'other')          unless test.metadata[:focus]
```

Comment:
what does `test.metadata` mean? why do you need this?

Reply:
It's for before(:each) block except some specific tests